### ISSUE TITLE
install jq failing because of "403 rate limit exceeded"

### ISSUE THREAD

**Issue by sirtimbly** [CircleCI-Public/jira-connect-orb#36]
### Orb version

1.1.2

### What happened

Build fails during `install jq` because of an API rate limit. 

```
curl: (22) The requested URL returned error: 403 rate limit exceeded
```

My guess is that everyone who uses this orb is counting towards the rate limit for for `api.github.com`.

### Expected behavior

Build should succeed, `jq` install script should not be hitting that API, maybe just lock the version for safety anyway?

**Comment by caleb15** [comment]
See https://github.com/CircleCI-Public/jq-orb/pull/11  - this has been fixed in jq,

**Comment by lokst** [comment]
We'll be looking into addressing this quickly

**Comment by caleb15** [comment]
@lokst you can fix it quickly by upgrading jq orb to v2.0.0.

From the issue I linked:
> You're welcome. The orb didn't actually publish until this morning. The new orb that won't have this GitHub API issue was published as v2.0.0.

**Comment by lokst** [comment]
`circleci/jira@1.1.3` which contains the fix is now available :tada:
